Fix/data-transform (#1398)
hadley/r4ds: data-transform.qmd

```

In both views, the variables names are followed by abbreviations that tell you the type of each variable: `<int>` is short for integer, `<dbl>` is short for double (aka real numbers), `<chr>` for character (aka strings), and `<dttm>` for date-time.

> These are important because the operations you can perform on a column depend so much on its "type", and these types are used to organize the chapters in the Transform part of the book.

Reviewer: ```suggestion
These are important because the operations you can perform on a column depend so much on its "type".
```

```

This "works", in the sense that it doesn't throw an error, but it doesn't do what you want because `|` first checks the condition `month == 1` and then checks the condition `2`, which is not a sensible condition to check.

> We'll learn more about what's happening here and why in @sec-order-of-operations-logicals.

Reviewer: ```suggestion
We'll learn more about what's happening here and why in @sec-boolean-operations.
```

## Columns

> There are four important verbs that affect the columns without changing the rows: `mutate()` creates new columns that are derived from the existing columns; `select()` changes which columns are present; `rename()` changes the names of the columns; and `relocate()` changes the positions of the columns.

Reviewer: ```suggestion
There are four important verbs that affect the columns without changing the rows: `mutate()` creates new columns that are derived from the existing columns, `select()` changes which columns are present, `rename()` changes the names of the columns, and `relocate()` changes the positions of the columns.
```
